s127/models/information_type.py

```diff
@@ -426,6 +489,69 @@ class Meta:
         verbose_name_plural = "Contact Addresses"
 
 
+class Radiocommunications(s100.models.ComplexAttributeType):
+    """
+    Detailed radiocommunications description with channels, frequencies, preferences and time schedules.
+
+    https://github.com/betagouv/SPPNautInterface/issues/273
+    https://github.com/betagouv/SPPNautInterface/issues/240
+    https://github.com/betagouv/SPPNautInterface/issues/241
```

Comment:
Note to self: Review the issues.

Also, we tend to put those issues in comments, not docstrings.